tests/attestation_plugin/plugin/tests.c

```diff
@@ -552,6 +555,7 @@ void verify_sgx_evidence(
         report_body = report->report;
         report_body_size = report->report_size;
         is_local = true;
+        OE_UNUSED(sgx_endorsements);
```

Comment:
I don't think you need this as long as sgx_endorsements is referred at least once in this function.

Reply:
Got it, thanks.